Backend/src/main/java/com/oops/backend/Backend/requests/CancelSolverRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.oops.backend.Backend.requests;
+
+public class CancelSolverRequest {
+    private String problemID;
```

Comment:
The API spec was slightly wrong, it is changed now, should be "ProblemID and UserID.